src/Fhir.Anonymizer.FunctionalTests/TestResources/patient-basic-target.json

```diff
@@ -58,6 +62,21 @@
       "period": {
         "start": "2000-02-11"
       }
+    },
+    {
+      "use": "home",
+      "type": "both",
+      "text": "123 Test St Test1, Test2, Test3 1234",
+      "line": [
+        "123 Test St"
+      ],
+      "city": "KeepTest",
+      "district": "TestRegion",
```

Comment:
It seems this target file contains sensitive information like extensions, address?